.github/workflows/build-and-release-single-package.yml

```diff
@@ -165,3 +165,10 @@ jobs:
         shell: bash
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Ensure Taipy release is marked as latest
+        run: |
+           gh release edit ${{needs.fetch-versions.outputs.config_VERSION}} --latest
```

Comment:
I believe we can use the `config_VERSION` or any other `{subpackage}_VERSION` right?

Anyway, the `needs.fetch-versions.outputs.VERSION` and the `needs.fetch-versions.outputs.NEW_VERSION` are redundant on line 30-31 right?

Reply:
>I believe we can use the config_VERSION or any other {subpackage}_VERSION right?

It should be always Taipy Version, because thats the one we want to make latest. I made a typo.

>Anyway, the needs.fetch-versions.outputs.VERSION and the needs.fetch-versions.outputs.NEW_VERSION are redundant on line 30-31 right?
Right. Good catch.